dopamine/utils/threading_utils.py

```diff
@@ -16,11 +16,23 @@
 
 The `local_attributes` class decorator defines a custom getter, setter and
 deleter for each specified attribute. These getter, setter and deleter actually
-wrap internal attributes that are thread specific.
+wrap internal attributes that are specific to the id of the thread they're
+accessed from.  We call those attributes "thread local".
 
-Each attribute has a callable default value that initializes the local value the
-first time they are called in a thread. To set these default values, use the
-`initialize_local_attributes` helper of this module.
+Each attribute has a callable default value that initializes a thread-local
+variable when it is not bound to a class at the time a caller tries to access
```

Comment:
would `object` be more appropriate here?

Reply:
You're totally right.  Fixed.

Reply:
I've decided that "bind" mostly just makes sense when talking about local or global namespaces.  When talking about module, class, or object namespaces, referring to presence in each's `__dict__` is clearer.  Thinking in terms of `__dict__`s has also helped me better understand how `local_attributes` works.  I've gone back and changed language and added some more descriptions of effects.

Thank you for your close reading.

Reply:
That's a good approach, I think, it's clearer.